### PR TITLE
Add docs for docs.jupiterone.io

### PR DESCRIPTION
**Pull request by fomentia**
Tried to publish and an error told me that I needed to write some docs for docs.jupiterone.io. So cool.

### CODE REVIEW

File: docs/jupiterone-io/jupiter-integration-whitehat.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| Scan Type                | `whitehat_scan` : `Service`                |
+| CVE                      | `cve` : `Weakness`                         |
+| Vulnerability            | `whitehat_vulnerability` : `Vulnerability` |
+| Finding                  | `whitehat_finding` : `Vulnerability`       |
+
+## Relationships
+
+The following relationships are created/mapped:
+
+### Basic relationships within the integration instance account/resources
```
Comment (aiwilliams) [inline docs/jupiterone-io/jupiter-integration-whitehat.md]: Could you try running `yarn lint` on this project? I think it should complain about the length of this line. We need to update the `managed-integration-example` docs this comes from to drop this line altogether.

File: docs/jupiterone-io/jupiter-integration-whitehat.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+|
```
Comment (aiwilliams) [inline docs/jupiterone-io/jupiter-integration-whitehat.md]: I think this table will not render properly. Compare https://docs.jupiterone.io/en/latest/docs/integrations/aws/AWS.html#relationships, which needs to be fixed, with https://docs.jupiterone.io/en/latest/docs/integrations/onelogin/jupiter-integration-onelogin.html#relationships. The fix probably needs to be back ported into the `managed-integration-example` as well, for the table below also.

**Comment by fomentia** [comment]
I also bumped the SDK peer dependency to v20. Is that ok to do without making extra changes @aiwilliams ?

**Comment by aiwilliams** [comment]
> I also bumped the SDK peer dependency to v20. Is that ok to do without making extra changes @aiwilliams ?

I think you'll need to fix the `tools/execute.ts` script. Could you look at fixing the linter to catch that file being broken?

**Comment by fomentia** [comment]
@aiwilliams Turns out I hadn't upgraded the SDK dev dep. Whoops! `tools/execute.ts` is updated now.

**Pull request merged**